Edit README_with Java and Kotlin (#25)
euphony-io/euphony: GETTING_STARTED.md

## in Transmitter

---

### Generate sound waves and transmits though a microphone.

1. **Create EuTxManager type object which turns data into sound waves.**
- Java

```java
EuTxManager mTxManager = new EuTxManager();
```

- Kotlin

```kotlin
val mTxManager = EuTxManager()
```

2. **Use euInitTransmit method, enter the data you want to transmit.**
- Java

```java
mTxManager.euInitTransmit("Hello, Euphony");
```

- Kotlin

```kotlin
mTxManager.euInitTransmit("Hello, Euphony")
```

3. **With process method, set how many times to repeat the sound(data). (-1 is infinite)**
- Java

```java
mTxManager.process(-1);
```

- Kotlin

```kotlin
mTxManager.process(-1)
```

## in Receiver

---

### Part of receiving sound waves.

> **Requires permission of RECORD_AUDIO**

- Write in AndroidManifest.xml

```java
<uses-permission android:name="android.permission.RECORD_AUDIO" />
```

1. **Create EuRxManager object that receives sound waves as data**
- Java

```java
EuRxManager mRxManager = new EuRxManager();
```

- Kotlin

```kotlin
val mRxManager = EuRxManager()
```

2. **Listener settings that receive and process sound waves**
- Java

```java
mRxManager.setAcousticSensor(new AcousticSensor() {

> 		@Override

Reviewer: It's better to update indent for readability 😄

# Getting Started

> Euphony is very easy to use
> 
> This consists of two parts.
> 
> 1) Transmitter
>  - Generate sound waves and transmits through a microphone.
> 2) Receiver
>  - Listen to sound waves and receive data.

Reviewer 2: ```suggestion
Euphony is a library that transmits data using critical bandwidths of the tonal frequency inaudible to humans.  
Since the API is written in Java, users can use it in any language running on JVM like Java or Kotlin.

This consists of two parts.

1) Transmitter
 - Generate sound waves and transmits through a speaker.
2) Receiver
 - Listen to sound waves and receive data through a MIC.
```

1. **Create EuTxManager type object which turns data into sound waves.**
- Java

```java
EuTxManager mTxManager = new EuTxManager();
```

- Kotlin

```kotlin
val mTxManager = EuTxManager()
```

2. **Use euInitTransmit method, enter the data you want to transmit.**
- Java

```java
mTxManager.euInitTransmit("Hello, Euphony");
```

- Kotlin

```kotlin
mTxManager.euInitTransmit("Hello, Euphony")
```

3. **With process method, set how many times to repeat the sound(data). (-1 is infinite)**
- Java

```java
mTxManager.process(-1);
```

- Kotlin

```kotlin
mTxManager.process(-1)
```

## in Receiver

---

### Part of receiving sound waves.

> **Requires permission of RECORD_AUDIO**

- Write in AndroidManifest.xml

```xml
<uses-permission android:name="android.permission.RECORD_AUDIO" />
```

1. **Create EuRxManager object that receives sound waves as data**
- Java

```java
EuRxManager mRxManager = new EuRxManager();
```

- Kotlin

```kotlin
val mRxManager = EuRxManager()
```

2. **Listener settings that receive and process sound waves**
- Java

```java
mRxManager.setAcousticSensor(new AcousticSensor() {
	@Override
	public void notify(String letters) {
      //when data is received

>     }

Reviewer 2: Could you check indent one more time?

> mRxManager.setAcousticSensor(new AcousticSensor() {
> 	@Override
> 	public void notify(String letters) {
> 		//when data is received
>     }
>  });
> ```

Reviewer 2: ```suggestion
mRxManager.setAcousticSensor(new AcousticSensor() {
	@Override
	public void notify(String letters) {
		// when data is received
	}
});
```
```

>  });

Reviewer: This code block indent is not fixed yet. Would you check this?

- Kotlin

```kotlin
mRxManager.acousticSensor = AcousticSensor { letters ->

>     //when data is received

Reviewer 3: I think the indentation interval should be the same as the 90th line.
On the 90th line, it seems to use the tab button.

# Getting Started

> 

Reviewer 2: ```suggestion
English | [한글](GETTING_STARTED_KR.md)
  
```
